source/installer.py

```diff
@@ -237,10 +238,53 @@ def removeOldProgramFiles(destPath):
 	"URLInfoAbout":versionInfo.url,
 }
 
-def registerInstallation(installDir,startMenuFolder,shouldCreateDesktopShortcut,startOnLogonScreen,configInLocalAppData=False):
-	with winreg.CreateKeyEx(winreg.HKEY_LOCAL_MACHINE,"SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Uninstall\\NVDA",0,winreg.KEY_WRITE) as k:
-		for name,value in uninstallerRegInfo.items(): 
-			winreg.SetValueEx(k,name,None,winreg.REG_SZ,value.format(installDir=installDir))
+
+def getDirectorySize(path: str) -> int:
+	"""Calculates the size of a directory in bytes.
+	"""
+	total = 0
+	with os.scandir(path) as iterator:

```

Comment:
Rather than using recursion on `getDirectorySize`, it might be better to use `glob` which returns files within folders recursively.

Reply:
Thanks for the suggestion. I just tried glob, but there are two disadvantages:

1. Glob simply returns files and folders, so we need to call os.path.isfile on every node
2. It is noticeably slower